core/templates/pages/topics-and-skills-dashboard-page/create-new-skill.modal.controller.ts

```diff
@@ -0,0 +1,103 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+/**
+ * @fileoverview Controller for CreateNewSkillModal.
+ */
+
+require('domain/utilities/url-interpolation.service.ts');
+require('services/context.service.ts');
+require('services/image-local-storage.service.ts');
+
+angular.module('oppia').controller('CreateNewSkillModal', [
+  '$scope', '$uibModalInstance', 'ContextService', 'ImageLocalStorageService',
+  'RubricObjectFactory', 'SkillCreationService', 'SkillObjectFactory',
+  'MAX_CHARS_IN_SKILL_DESCRIPTION', 'SKILL_DESCRIPTION_STATUS_VALUES',
+  'SKILL_DIFFICULTIES',
+  function($scope, $uibModalInstance, ContextService, ImageLocalStorageService,
+      RubricObjectFactory, SkillCreationService, SkillObjectFactory,
+      MAX_CHARS_IN_SKILL_DESCRIPTION, SKILL_DESCRIPTION_STATUS_VALUES,
+      SKILL_DIFFICULTIES) {
+    var rubrics = [
+      RubricObjectFactory.create(SKILL_DIFFICULTIES[0], []),
+      RubricObjectFactory.create(SKILL_DIFFICULTIES[1], ['']),
+      RubricObjectFactory.create(SKILL_DIFFICULTIES[2], [])];
+    ContextService.setImageSaveDestinationToLocalStorage();
+    $scope.newSkillDescription = '';
+    $scope.rubrics = rubrics;
+    $scope.errorMsg = '';
+    $scope.bindableDict = {
+      displayedConceptCardExplanation: ''
+    };
+    $scope.MAX_CHARS_IN_SKILL_DESCRIPTION = (
+      MAX_CHARS_IN_SKILL_DESCRIPTION);
+    var newExplanationObject = null;
+
+    $scope.$watch('newSkillDescription', function() {
+      if (
+        SkillCreationService.getSkillDescriptionStatus() !==
+                SKILL_DESCRIPTION_STATUS_VALUES.STATUS_DISABLED) {
```

Comment:
Shouldn't this be indented by only 2 from the previous line?

Reply:
Done.